### PR TITLE
adding cartodb demo

### PR DESCRIPTION
**Pull request by ohasselblad**
@etpinard here's my notebook for cartodb

### CODE REVIEW

**Comment by etpinard** [comment]
@ohasselblad First of all, thank you very much for doing this. Your notebook looks great.

Only a few modifications need to be made before merging.

File: notebooks/cartodb/cartodb.ipynb [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+      "\n",
+      "By selecting the Torque in the [Visualization Wizard](http://docs.cartodb.com/cartodb-editor.html#wizards) you can get an animated map of the earthquakes over time. Make sure to select the `time` column in the wizard. By clicking on the `CSS` tab, you can customize your map further. Copy &amp; Past the CartoCSS below the map to reproduce it's style."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from IPython.display import HTML\n",
+      "HTML('<iframe width=100% height=520 frameborder=0 src=http://andye.cartodb.com/viz/e44ac140-b8ad-11e4-b156-0e4fddd5de28/embed_map allowfullscreen webkitallowfullscreen mozallowfullscreen oallowfullscreen msallowfullscreen></iframe>')"
```
Comment (etpinard) [inline notebooks/cartodb/cartodb.ipynb]: plot.ly needs https support. You'll have to change the `src` to https://andye.cartodb ... to make the iframe display correctly.

File: notebooks/references.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
         "networkx",
         "make_subplots",
         "basemap",
-        "collaborate"
+        "collaborate",
+        "cartodb"
```
Comment (etpinard) [inline notebooks/references.json]: I'm going to ask you to put "cartodb" at the top of the list here. (Sorry about that, this part is poorly designed).

File: notebooks/cartodb/config.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "title": "CartoDB and Plotly",
+    "title_short": "CartoDB + Plotly",
+    "meta_description": "CartoDB and Plotly mashup using the Plotly, CartoDB, Pandas, and IPython Notebooks",
+    "cells": [0, -1],
+    "relative_url": "",
```
Comment (etpinard) [inline notebooks/cartodb/config.json]: You'll need to choose a relative url e.g. "cartodb".

In this case, your notebook would be hosted at https://plot.ly/ipython-notebooks/cartodb

**Comment by etpinard** [comment]
Once those three modifications are made, run `$ make run nb=cartodb` and `$ make run nb=cartodb` and push.

**Comment by ohasselblad** [comment]
Brilliant, thanks for the suggestions. Let me know if there's anything else.

**Comment by etpinard** [comment]
Thank you very much. Your notebook will be deployed to plot.ly in the next hour.

**Pull request merged**